packages/react/src/components/LightboxMediaViewer/LightboxMediaViewer.js

```diff
@@ -22,65 +23,29 @@ const { prefix } = settings;
 /**
  * LightboxMediaViewer Component
  *
- * @param {object} props props object
- * @param {string} props.title LightboxMediaViewer media title
- * @param {string} props.copy LightboxMediaViewer media short description
- * @param {string} props.image LightboxMediaViewer responsive image object
- * @param {boolean} props.open sets whether the modal is open/close
- * @param {Function} props.onClose do something on close in addition, return false to completely replace
+ * @param {object} modalProps props object
+ * @param {string} modalProps.title LightboxMediaViewer media title
+ * @param {string} modalProps.copy LightboxMediaViewer media short description
+ * @param {string} modalProps.image LightboxMediaViewer responsive image object
+ * @param {boolean} modalProps.open sets whether the modal is open/close
  * @returns {*} JSX Object
  */
-const LightboxMediaViewer = ({ title, copy, image, open, onClose }) => {
-  /**
-   * Set default image
-   *
-   * @param {*} image responsive image object
-   * @returns {string} default image url
-   */
-  const defaultImage = image => {
-    return image.uri.md
-      ? image.uri.md
-      : image.uri.lg
-      ? image.uri.lg
-      : image.uri.sm;
-  };
-
-  if (!image.uri.lg) {
-    return null;
-  }
+const LightboxMediaViewer = ({ title, images, copy, ...modalProps }) => {
   return featureFlag(
     DDS_LIGHTBOX_MEDIA_VIEWER,
     <section
       data-autoid={`${stablePrefix}--lightbox-media-viewer`}
       className={`${prefix}--lightbox-media-viewer`}>
-      <ExpressiveModal open={open} fullwidth={true} onClose={onClose}>
+      <ExpressiveModal fullwidth={true} {...modalProps}>
         <ModalBody>
           <div className={`${prefix}--lightbox-media-viewer__container`}>
             <div className={`${prefix}--lightbox-media-viewer__row`}>
               <div
                 data-autoid={`${stablePrefix}--lightbox-media-viewer__image`}
                 className={`${prefix}--lightbox-media-viewer__image`}>
-                <picture>
-                  {image.uri.lg && (
-                    <source
-                      media="(min-width: 672px)"
-                      srcSet={`${image.uri.lg}`}
-                    />
-                  )}
-                  {image.uri.md && (
-                    <source
-                      media="(min-width: 320px)"
-                      srcSet={`${image.uri.md}`}
-                    />
-                  )}
-                  {image.uri.sm && (
-                    <source
-                      media="(max-width: 320px)"
-                      srcSet={`${image.uri.sm}`}
-                    />
-                  )}
-                  <img src={defaultImage(image)} alt={image.alt}></img>
-                </picture>
+                {images.map((image, index) => {
+                  return <Image {...image} key={index} />;
+                })}
```

Comment:
I don't think the loop is necessary. The `Image` component handles all of that for us. We just want to render one image here.

Reply:
Okay, looked Image component in `packages/react` that make sense will update without iteration.